ecommerce/settings/production.py

```diff
@@ -118,3 +118,6 @@ def get_env_setting(setting):
 CORS_ALLOW_HEADERS = corsheaders_default_headers + (
     'use-jwt-cookie',
 )
+
+# To customize the seat name
+CUSTOM_COURSE_SEAT_NAME = config_from_yaml.get('CUSTOM_COURSE_SEAT_NAME', '')
```

Comment:
Did you check that "config_from_yaml" works as expected? @Jacatove

Reply:
Yes, I checked it, but I also found out that if one configures the variable in the .yml, there is no need to define it neither in settings/production nor settings/base. So, Should I remove those lines?. @Squirrel18

Reply:
I don't think so, let's just leave it as it is to keep track of the change. @Jacatove